templates/log4j.properties.j2

```diff
@@ -0,0 +1,58 @@
+# Define some default values that can be overridden by system properties
+zookeeper.root.logger=INFO, CONSOLE, ROLLINGFILE
+zookeeper.console.threshold=INFO
+zookeeper.log.dir=.
+zookeeper.log.file=zookeeper.log
+zookeeper.log.threshold=DEBUG
+zookeeper.tracelog.dir=.
```

Comment:
I assume that it takes configuration of log directory from https://github.com/AnsibleShipyard/ansible-zookeeper/blob/master/templates/zoo.cfg.j2#L3 ?